gamemodes/terrortown/gamemode/client/cl_help/populate_appearance.lua

```diff
@@ -0,0 +1,554 @@
+
+	form:MakeSlider({
+		label = "label_crosshair_opacity",
+		convar = "ttt_crosshair_opacity",
+		min = 0,
+		max = 1,
+		decimal = 1,
+		master = crossEnb
+	})
+
+	form:MakeSlider({
+		label = "label_crosshair_ironsight_opacity",
+		convar = "ttt_ironsights_crosshair_opacity",
+		min = 0,
+		max = 1,
+		decimal = 1,
+		master = crossEnb
+	})
+
+	form:MakeSlider({
+		label = "label_crosshair_size",
+		convar = "ttt_crosshair_size",
+		min = 0.1,
+		max = 3,
+		decimal = 1,
+		master = crossEnb
+	})
+
+	form:MakeSlider({
+		label = "label_crosshair_thickness",
+		convar = "ttt_crosshair_thickness",
+		min = 1,
+		max = 10,
+		decimal = 0,
+		master = crossEnb
+	})
+
+	form:MakeSlider({
+		label = "label_crosshair_thickness_outline",
+		convar = "ttt_crosshair_outlinethickness",
+		min = 0,
+		max = 5,
+		decimal = 0,
+		master = crossEnb
+	})
+
+	form:MakeCheckBox({
+		label = "label_crosshair_static_enable",
+		convar = "ttt_crosshair_static",
+		master = crossEnb
+	})
+
+	form:MakeCheckBox({
+		label = "label_crosshair_dot_enable",
+		convar = "ttt_crosshair_dot",
+		master = crossEnb
+	})
+
+	form:MakeCheckBox({
+		label = "label_crosshair_scale_enable",
+		convar = "ttt_crosshair_weaponscale",
+		master = crossEnb
+	})
+
+	form:MakeCheckBox({
+		label = "label_crosshair_ironsight_low_enabled",
+		convar = "ttt_ironsights_lowered",
+		master = crossEnb
+	})
+end
+
+local function PopulateDamagePanel(parent)
+	local form = CreateForm(parent, "header_damage_indicator")
+
+	form:MakeHelp({
+		label = "help_damage_indicator_desc"
+	})
+
+	-- store the reference to the checkbox in a variable
+	-- because the other settings are enabled based on
+	-- the state of this checkbox
+	local dmgEnb = form:MakeCheckBox({
+		label = "label_damage_indicator_enable",
+		convar = "ttt_dmgindicator_enable"
+	})
+
+	form:MakeComboBox({
+		label = "label_damage_indicator_mode",
+		convar = "ttt_dmgindicator_mode",
+		choices = DMGINDICATOR.GetThemeNames(),
+		master = dmgEnb
+	})
+
+	form:MakeSlider({
+		label = "label_damage_indicator_duration",
+		convar = "ttt_dmgindicator_duration",
+		min = 0,
+		max = 30,
+		decimal = 2,
+		master = dmgEnb
+	})
+
+	form:MakeSlider({
+		label = "label_damage_indicator_maxdamage",
+		convar = "ttt_dmgindicator_maxdamage",
+		min = 0,
+		max = 100,
+		decimal = 1,
+		master = dmgEnb
+	})
+
+	form:MakeSlider({
+		label = "label_damage_indicator_maxalpha",
+		convar = "ttt_dmgindicator_maxalpha",
+		min = 0,
+		max = 255,
+		decimal = 0,
+		master = dmgEnb
+	})
+end
+
+local function PopulatePerformancePanel(parent)
+	local form = CreateForm(parent, "header_performance_settings")
+
+	form:MakeCheckBox({
+		label = "label_performance_halo_enable",
+		convar = "ttt_entity_draw_halo"
+	})
+
+	form:MakeCheckBox({
+		label = "label_performance_spec_outline_enable",
+		convar = "ttt2_enable_spectatorsoutline"
+	})
+
+	form:MakeCheckBox({
+		label = "label_performance_ohicon_enable",
+		convar = "ttt2_enable_overheadicons"
+	})
+end
```

Comment:
Hm, for me it has no inpact on performance. And honestly, it is something I wouldn't search for in performance. Actually gameplay could be the best place for this since it has a huge influence on the gameplay.
But in my opinion it bbelongs to the vskin